Makefile

```diff
@@ -8,10 +8,14 @@ CXXFLAGS += -std=c++11 -I src/gen/ -I src/gen/proto/
 
 all: buf 
 
-buf:	buf-clean
+buf: buf-clean
 	@mkdir -p src/gen
 	buf generate buf.build/viamrobotics/goutils --template buf.gen.yaml
 	buf generate buf.build/googleapis/googleapis --template buf.gen.yaml --path google/rpc --path google/api 
-	buf generate buf.build/viamrobotics/api --template buf.gen.yaml
+	buf generate buf.build/viamrobotics/api --template buf.gen.yaml --path common,component,robot,service
```

Comment:
I think we should add `module` to the `--path` flag as well, we'll be relying on it for the modular resources wrappers.

Reply:
Yup! I don't think `module` was in when I made this change, so I'll add it in